client/onboarding-prototype/steps/loading.tsx

```diff
@@ -58,7 +56,9 @@ const Loading: React.FC = () => {
 			? addQueryArgs( connectUrl, {
 					progressive: fromDotNotation( data ),
 			  } )
-			: connectUrl;
+			: addQueryArgs( connectUrl, {
+					prefill: fromDotNotation( data ),
+			  } );
 		window.location.href = resultUrl;
```

Comment:
__Nitpick__: We can avoid writing`addQueryArgs` twice using a dynamic key. Something like:
```js
const resultUrl = addQueryArgs(connectUrl, {
  [isElegible ? 'prefill' : 'progressive']: fromDotNotation(data),
});

```

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/commit/9d784e9e18ba42be752b58303dad303fba8286fe!